addons/l10n_zm/models/template_zm.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models
+from odoo.addons.account.models.chart_template import template
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = "account.chart.template"
+
+    @template('zm')
+    def _get_zm_template_data(self):
+        return {
+            'code_digits': 7,
+            'property_account_receivable_id': '8000000',
+            'property_account_payable_id': '9000000',
+            'property_account_expense_categ_id': '3800000',
+            'property_account_income_categ_id': '1000000',
+        }
+
+    @template('zm', 'res.company')
+    def _get_zm_res_company(self):
+        return {
+            self.env.company.id: {
+                'anglo_saxon_accounting': True,
+                'account_fiscal_country_id': 'base.zm',
+                'bank_account_code_prefix': '840000',
+                'cash_account_code_prefix': '840000',
+                'transfer_account_code_prefix': '840000',
+                'income_currency_exchange_account_id': '4210000',
+                'expense_currency_exchange_account_id': '4210000',
+                'account_default_pos_receivable_account_id': '8100000',
+                'account_journal_early_pay_discount_loss_account_id': '3550000',
+                'account_journal_early_pay_discount_gain_account_id': '2700000',
+                'deferred_expense_account_id': '8900000',
+                'deferred_revenue_account_id': '9900000',
```

Comment:
Shouldn't those be only in Enterprise ?

Reply:
Since https://github.com/odoo/odoo/pull/138937 and https://github.com/odoo/enterprise/commit/31c95089c718227de3652173486f11215cd75b7c it is possible to define the defered accounts in odoo, making it here allows us not to create a template_zm.py file in enterprise which would just do that.
It's better to leave them here i believe :+1:

Reply:
Totally fine by me, but then no need to put them again in Enterprise ?
https://github.com/odoo/enterprise/pull/54707/files#diff-0d1d550e8d8cd78f95e33bf3c0437e8e9750e00f548fb5686789c66947dcb8e8R14-R15

Reply:
Indeed. This is a leftover i forgot to remove. ^^'